docker-compose.yml

```diff
@@ -87,6 +87,7 @@ services:
       - "BALANCE_SERVER=https://balance-staging.mercury.basicattentiontoken.org"
       - TOKEN_LIST
       - UPHOLD_ACCESS_TOKEN
+      - FEATURE_ORDERS=true
```

Comment:
If this set to true will it honor the value in the environment?

Reply:
reverted in next commit.